ml-lifecycle/feature_store/README.md

```diff
@@ -1,4 +1,68 @@
-# feature-store
-SageMaker feature store
+# Amazon SageMaker Feature Store
+## Using Health Record Data to Detect Heart Failure with SageMaker Feature Store
 
+## Contents
+1. [Background](#Background)
+1. [Prerequisites](#Prereqs)
+1. [Data](#Data)
+1. [Approach](#Approach)
+1. [Clean Up](#Clean-Up)
+1. [Other Resources](#Other-Resources)
 
+---
+
+# Background
+
+SageMaker FeatureStore makes it easy to create and manage curated features for machine learning (ML) development. It serves as the single source of truth to store, retrieve, remove, track, share, discover, and control access to features. SageMaker Feature Store enables data ingestion via a high TPS API and data consumption via the online and offline stores.
```

Comment:
Be consistent with `Feature Store`- this looks like the only instance of `FeatureStore`.